### PR TITLE
feat(docs): add home and playground links to sidebar navigation

### PR DESCRIPTION
**Pull request by tmorehouse**
### Describe the PR

adds homepage and playground links to sidebar navigation

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Enhancement
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** (check one)

- [x] No
- [ ] Yes (please describe)

**The PR fulfills these requirements:**

- [] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples, fix typos`, `chore: fix typo in README`, etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [ ] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [ ] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (Does it affect screen reader users or keyboard only users? Clickable items should be in the tab index, etc.)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: docs/components/sidebar.vue [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <template>
   <b-collapse id="bd-docs-nav" class="bd-links" tag="nav" is-nav>
+    <b-link to="/" exact router-tag="div" active-class="active">
+      <b-link to="/" exact class="bd-toc-link" active-class="">
+        BootstrapVue
```
Comment (jackmu95) [inline docs/components/sidebar.vue]: Maybe just "Home" would feit better here

Comment (tmorehouse) [reply docs/components/sidebar.vue]: I thought about that... but wasn't sure if that sounded too cheezy

Comment (jackmu95) [reply docs/components/sidebar.vue]: I don't think so. Makes it much more clear IMHO.

Comment (tmorehouse) [reply docs/components/sidebar.vue]: OK... lets change it to `Home`

**Pull request merged**